README.md

```diff
@@ -1,5 +1,29 @@
 # best-programming-club-2020-12-20
 
+## Kim's BYO-Assignment: Add Detailed Cart View ##
```

Comment:
This is great content to paste in the body of a pull request @kmbrlykm I copy-pasted it to this PR as an example